skaffold.yaml

```diff
@@ -12,6 +12,12 @@
 apiVersion: skaffold/v2beta1
 kind: Config
 build:
+  local:
+    useDockerCLI: false
+    useBuildkit: false
+    push: true
+  insecureRegistries:
```

Comment:
I think we should have 1/2 different profiles in Skaffold. Because as a developer I may not want to have a registry locally. In case of GKE workflow - this not work at all. And in case of Kind this is needed.